sdk/messaging/azwebpubsub/healthapi_client.go

```diff
@@ -0,0 +1,61 @@
+//go:build go1.18
+// +build go1.18
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// Code generated by Microsoft (R) AutoRest Code Generator. DO NOT EDIT.
+// Changes may cause incorrect behavior and will be lost if the code is regenerated.
+
+package azwebpubsub
+
+import (
+	"context"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/policy"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/runtime"
+	"net/http"
+)
+
+// HealthAPIClient contains the methods for the HealthAPI group.
+// Don't use this type directly, use a constructor function instead.
+type HealthAPIClient struct {
```

Comment:
I'm thinking this should just be `HealthClient`. @JeffreyRichter what do you think?

Reply:
Also, I don't see any constructors for this client. How is it used?

Reply:
Added ctors. Keep the naming as `HealthAPIClient` as it is the default one auto generated. Waiting for experts confirmation on the naming to update it if needed.

Reply:
Yes, HealthClient is better than having API in the name.